libpdbg/libpdbg.h

```diff
@@ -1459,8 +1459,8 @@ bool pdbg_context_short(void);
  * switching the backend in the same process. It clears/releases the 
  * existing dev tree (if any) children by children along with the root node.
  * 
- * Call this function very judiciously before ##pdbg_set_backend() if there is a
```

Comment:
Same as above commit

Reply:
This was also changed during writing unit test cases.